### DISCUSSION TITLE
Door / Window sensor and device type

### DISCUSSION THREAD

**Discussion by dbrb2** [Koenkk/zigbee2mqtt#24062] [Devices]
Hi, 
Apologies if this turns out to be RTFM - but I can't find the answer so far! 
I have added a load of Aqara door/window sensors to Zigbee2Mqtt, which itself was installed as an add-on for HA

This all worked fine, but these sensors all report themselves to HA as device type Door - I know I can override this in the UI, but is there a way to define it as a window before it is reported to HA? 

![image](https://github.com/user-attachments/assets/36d703eb-286a-436a-8ef0-23b124c65589)

**Comment by bidrohini** [comment]
Open your Zigbee2MQTT configuration file, typically located at /config/zigbee2mqtt/configuration.yaml within your Home Assistant setup. Find the sensor by its friendly_name or IEEE address. You can find this information in the Zigbee2MQTT web interface or the Home Assistant device list. Inside configuration.yaml, add an entry for the specific device where you override the name to "Window" or any custom name. I hope this will solve the issue you're facing. 
You can also take a look at this door garage controller project: https://www.pcbway.com/project/shareproject/Wi_Fi_Garage_Door_Controller_with_Dual_Sensor_Integration_9c40bece.html
